### PR TITLE
131 volume for tokens

### PR DESCRIPTION
**Pull request by vstavetski**
added volumes for tokens
updated REST api

### CODE REVIEW

File: rest-api/src/main/kotlin/fund/cyber/markets/rest/common/CrossConversion.kt [hunk 1]
```diff
@@ -88,15 +88,15 @@ class CrossConversion(
     }
 
     private fun calcInvert(ticker: Ticker): BigDecimal {
-        return BigDecimal(1).divide(ticker.price, 20, RoundingMode.HALF_EVEN)
+        return BigDecimal(1).divide(ticker.close, 20, RoundingMode.HALF_EVEN)
```
Comment (YodaMike) [inline rest-api/src/main/kotlin/fund/cyber/markets/rest/common/CrossConversion.kt]: remove magic number 20

File: rest-api/src/main/kotlin/fund/cyber/markets/rest/handler/HistoHandler.kt [hunk 2]
```diff
@@ -42,30 +45,35 @@ class HistoHandler(
...
+            val volumeQuote = volumeRepository.get(ticker.pair.quote, exchange, duration, timestamp)
+
             val tickerData = TickerData(
-                    ticker.timestampTo?.time!!,
-                    ticker.price,
-                    ticker.maxPrice!!,
-                    ticker.minPrice!!,
-                    ticker.baseAmount,
-                    ticker.quoteAmount
+                    ticker.timestampTo?.time!! / 1000,
```
Comment (YodaMike) [inline rest-api/src/main/kotlin/fund/cyber/markets/rest/handler/HistoHandler.kt]: remove magic number 1000

File: tickers/src/main/kotlin/fund/cyber/markets/tickers/service/TickerService.kt [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    ).apply { initTransactions() }
+
+    init {
+        consumer.subscribe(configuration.topicNamePattern)
+        consumerBackup.subscribe(listOf(configuration.tickersBackupTopicName))
+        seekToEnd()
+    }
+
+    fun poll(timeout: Long): ConsumerRecords<String, Trade> {
+        return consumer.poll(timeout / 2)
```
Comment (YodaMike) [inline tickers/src/main/kotlin/fund/cyber/markets/tickers/service/TickerService.kt]: remove magic number 2

File: tickers/src/main/kotlin/fund/cyber/markets/tickers/configuration/TickersConfiguration.kt [hunk 4]
```diff
@@ -12,55 +13,102 @@ import java.util.concurrent.TimeUnit
 import java.util.regex.Pattern
 
 private val tradesTopicNamePattern = Pattern.compile("TRADES-.*")
+private val numPartitions = 1
+private val replicationFactor: Short = 1
+private val MAX_POLL_RECORDS = 20000
```
Comment (YodaMike) [inline tickers/src/main/kotlin/fund/cyber/markets/tickers/configuration/TickersConfiguration.kt]: use 'const' keyword

File: rest-api/src/main/kotlin/fund/cyber/markets/rest/common/CrossConversion.kt (hunk 1 above)
Comment (vstavetski) [reply rest-api/src/main/kotlin/fund/cyber/markets/rest/common/CrossConversion.kt]: done

File: tickers/src/main/kotlin/fund/cyber/markets/tickers/service/TickerService.kt (hunk 3 above)
Comment (vstavetski) [reply tickers/src/main/kotlin/fund/cyber/markets/tickers/service/TickerService.kt]: done

File: tickers/src/main/kotlin/fund/cyber/markets/tickers/configuration/TickersConfiguration.kt (hunk 4 above)
Comment (vstavetski) [reply tickers/src/main/kotlin/fund/cyber/markets/tickers/configuration/TickersConfiguration.kt]: done

**Pull request merged**